src/base.iced

```diff
@@ -728,6 +765,13 @@ class OuterLink
 
     arr.push (!!@ignore_if_unsupported) if @ignore_if_unsupported?
 
+    # If an older client wants to make an outer link, they will get null in both fields.
```

Comment:
The problem is if a new client wants to make a first link and send null,null for hprev, the service has no way to distinguish this from an old client that didn't send anything, and the generated linkv2s won't have those params in either case (since they're only pushed when nonnull). That might be OK though. Here, it's distinguishing, since 0 seqno is invalid otherwise.

Reply:
Why not:

```iced
arr.push 0
arr.push (new Buffer [])
```

Reply:
Then we can change the client be sensitive to this distinction.

Reply:
I can do that, but what is the advantage? I was hoping to keep `hash=null implies first seqno` for consistency.

Reply:
you're right.  ok we can leave it.